### PR TITLE
updated email address in privacy-policy.component.html

### PR DESCRIPTION
**Pull request by paddybasi**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-2655

### Change description ###
Updated email address

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by ernestman28 — APPROVED** [review]

**Review by paul-graham — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by Mo-Lala84 — APPROVED** [review]
approved

File: test/integration/tests/utils/http.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+axios.defaults.headers.common['Content-Type'] = 'application/json';
+
+s2s.configure({
+  microservice,
+  s2sEndpointUrl,
+  s2sSecret
+});
+
+const httpsagent = tunnel.httpsOverHttp({
```
Comment (paddybasi) [inline test/integration/tests/utils/http.ts:28]: I know I probably wrote some of this code, but we no longer need a proxy tunnel

**Review by paddybasi — COMMENTED** [review]
Just a small suggestion

**Review by Mo-Lala84 — APPROVED** [review]
Lgtm

**Pull request merged**